backend/backends/f5tts_backend.py

```
@@ -0,0 +1,230 @@
+    empty_device_cache,
+    manual_seed,
+    combine_voice_prompts as _combine_voice_prompts,
+    model_load_progress,
+)
+
+logger = logging.getLogger(__name__)
+
+# Monkeypatch torchaudio.load to use soundfile instead of torchcodec
+# TorchAudio 2.6 removed legacy soundfile fallback and strictly requires
+# TorchCodec, which crashes without full FFmpeg DLLs on Windows.
+import torchaudio
```

Ruff (0.15.20)

[error] Module level import not at top of file

(E402)